data/DataHandler.php

```diff
@@ -0,0 +1,152 @@
+<?php
+//==================================
+// Class Definition of DataHandler
+//==================================
+
+require __DIR__.'/../utility/Equipment.php';
+require __DIR__.'/../utility/Room.php';
+
+define ('WAREHOUSE', 'warehouse');
+define("MAX_SPACE", 1000);
+
+class DataHandler {
+    
+    private array $room_list;
+    private array $equip_list;
+    
+    /**
+     * Constructor for the Handler.
+     * Initializes two arrays containing all equipment and all rooms.
+     * Automatically creates the warehouse.
+     */
+    public function __construct() {
+        $this->room_list = [WAREHOUSE => new Room(WAREHOUSE, MAX_SPACE)];
+        $this->equip_list = [];
+    }
+
+
+    
+    /************** Equipment ****************/
+    /**
+     * Adds an equipment to the equipment array. Will allow for overwritting if equipment shares an equipment_id;
+     * @param  Equipment $equip to be added
+     * @return bool
+     */
+    public function add_equipment(Equipment $new_equip) : bool {
+        if (!$this->room_list[WAREHOUSE]->add_equipment($new_equip)) {
+            return false;
+        }
```

Comment:
Why do you always add to the Warhouse ? Did u mean this to be the general_inventory?